.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,41 @@
+<!--- If what you want to file is not a bug, please go to [other_issues] -->
```

Comment:
Should we create at least a placeholder entry in the wiki and put the link here?